packages/kbn-esql-validation-autocomplete/src/definitions/functions.ts

```diff
@@ -924,6 +918,30 @@ export const evalFunctionsDefinitions: FunctionDefinition[] = [
       },
     ],
   },
+  {
+    name: 'mv_sort',
+    description: i18n.translate('kbn-esql-validation-autocomplete.esql.definitions.mvSortDoc', {
+      defaultMessage: 'Sorts a multivalue expression in lexicographical order.',
+    }),
+    signatures: [
+      {
+        params: [
+          { name: 'field', type: 'any' },
+          {
+            name: 'order',
+            type: 'string',
+            optional: true,
```

Comment:
Add `literalOnly: true` here to enable the literal validation

Reply:
I didn't do this because you can technically use a function to build the sort string. See https://github.com/elastic/kibana/pull/179874#issuecomment-2043650803